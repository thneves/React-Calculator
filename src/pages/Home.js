import React from 'react';
import '../styles/Home.css';

const Home = () => (
  <div className="home">
    <h1>Math Magicians!</h1>

    <h3>Welcome to my page!</h3>
    <p>
      This is a Calculator project using React! You Can check the source code on this
      <a href="https://github.com/thneves/React-Calculator">link.</a>
    </p>
  </div>
);

export default Home;
