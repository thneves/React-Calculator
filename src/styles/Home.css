.home {
  background-color: #a7bdbe;
  height: 100vh;
  display: flex;
  flex-direction: column;
  padding-left: 17%;
  font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
}

.home h3 {
  color: #042324;
  font-size: 26px;
  padding-top: 100px;
  margin-bottom: 35px;
}

.home p {
  color: #042324;
  font-size: 20px;
  margin-bottom: 35px;
}

.home p a {
  color: white;
  font-weight: bold;
  text-decoration: none;
}
