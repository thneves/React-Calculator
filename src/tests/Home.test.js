import React from 'react';
import TestRenderer from 'react-test-renderer';
import Home from '../pages/Home';

it('should render Home page component', () => {
  const home = TestRenderer
    .create(<Home />)
    .toJSON();
  expect(home).toMatchSnapshot();
});
