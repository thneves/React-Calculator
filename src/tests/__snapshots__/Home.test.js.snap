// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Home page component 1`] = `
<div
  className="home"
>
  <h3>
    Welcome to my page!
  </h3>
  <p>
    This is a Calculator project using React! You can check the source code on this
    <a
      href="https://github.com/thneves/React-Calculator"
    >
       link. 
    </a>
    Math is an interesting thing don't you think?
  </p>
  <p>
    My name is Thales. I'm a software developer, you can check my Github account
    <a
      href="https://github.com/thneves"
    >
       here.
    </a>
  </p>
</div>
`;
